### PR TITLE
Don't init JFrog registry for non-gradle builds

### PR DESCRIPTION
**Pull request by stophi-dev**

### CODE REVIEW

**Comment by stefancplace** [comment]
Should we bump the version of cplace-asc ? Or are you going to do it manually after merging this PR?

**Review by stefancplace — APPROVED** [review]
Tested with node@12 in main and pp -> works as expected

File: package.json [hunk 1]
```diff
@@ -28,9 +28,9 @@
   "author": "collaboration Factory AG",
   "license": "ISC",
   "dependencies": {
-    "@openapitools/openapi-generator-cli": "2.1.26",
+    "@openapitools/openapi-generator-cli": "^2.4.26",
```
Comment (stefancplace) [inline package.json:31]: I`m not sure, maybe we should switch to pinpoint versions? (After all these hijacked npm packages)

**Comment by stophi-dev** [comment]
Not needed anymore after https://github.com/collaborationFactory/cplace-asc/pull/69.
The call to `gradlew` is removed and building the assets works again without `gradlew` being present in the main repository.

**Pull request closed without merging**